src/gen/local_lm.py

```diff
@@ -0,0 +1,57 @@
+import torch
+from transformers import (
+    AutoModelForCausalLM,
+    BitsAndBytesConfig
+)
+from alignment.model_utils import get_tokenizer
+
+def get_model(model_id, load_in_8bit, load_in_4bit, model_args, data_args):
+    """
+    get_model instantiates and return fine-tuned language model and tokenzier.
+
+    arguments:
+    model_args -- ModelArguments obtained from H4ArgumentParser
+    data_args -- DataArguments obtained from H4ArgumentParser
+    """
+    quantization_config = BitsAndBytesConfig(load_in_8bit=load_in_8bit, load_in_4bit=load_in_4bit)
+
+    tokenizer = get_tokenizer(model_args, data_args)
+    tokenizer.padding_side = 'left'
```

Comment:
Should be done within `get_tokenizer()` function itself and should be accompanied with a comment.

Reply:
I assume you are pointing out `tokenizer.padding_side = 'left'` only here, right?

Reply:
Yeah